source/about/mattermost-v10-changelog.md

```diff
@@ -1,12 +1,88 @@
+ - Added Shared Channels administration page to the System Console.
+ - Added a team selector to accept connection invite flow in Shared Workspaces.
+ - Restricted activation and deactivation of LDAP-managed users through both the System Admin Console UI and Mattermost API.
+ - Export/import improvements: added the ability to export all user preferences and flagged posts.
+ - Increased timeouts to fetch cluster logs.
+ - Improved log messages for cluster communication.
+ - Information about deleted rows from the Data Retention job are now logged.
+ - License details to logs are now emitted when added or removed.
+ - Added a new mmctl command, ``mmctl post delete <post-id>``, in order to permanently delete a post and its attachments.
+
+#### Performance
+ - Added metrics to prometheus to check the mobile versions for each session daily.
+ - Improved the performance of LDAP sync jobs when group-contained teams and channels are used.
+ - Added minor improvements to notification metrics.
+ - Added minor improvements to mobile push notifications.
+
+### Bug Fixes
+ - Fixed an issue with email notifications using 24-hour timestamps by default.
+ - Fixed an issue where bots were not ignored when counting deactivated accounts for statistics.
+ - Fixed an issue where drafts didn’t allow scrolling if the user had many drafts.
+ - Fixed an issue that caused Javascript errors in the System Console.
+ - Fixed racy use of session in ``NewWebConn``.
+ - Fixed a race condition that would happen after a server start if ``EnableTesting`` was enabled.
+ - Fixed an issue where no error message was shown when replying to a deleted post from the draft screen.
+ - Fixed an issue where the check icons were missing from the Sort and Show options in the Direct Messages tab, and the Sort tab of the Channels tab.
+ - Fixed desyncing issues with unreads between the team sidebar and the title bar.
+ - Fixed an issue with message export file attachments with dedicated filestore.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added a new configuration setting ``EnableAPIPostDeletion`` in order to enable/disable post deletion. This configuration setting does not need to be enabled when running mmctl in local mode.
+    - Added ``EnableDesktopLandingPage`` to allow the desktop app landing page to be disabled.
+ - Under ``NativeAppSettings`` in ``config.json``:
+     - Added a configuration setting ``MobileExternalBrowser`` that tells the Mobile app to perform SSO Authentication using the external default browser.
+
+### Go Version
+ - v10.2 is built with Go ``v1.22.6``.
+
+### Known Issues
+ - Shared Channels: Direct Messages are not supported.
```

Comment:
Are we sure about this?  I have tested direct messages locally.

Reply:
@sbishel shared this here https://hub.mattermost.com/abc/pl/jwb6hc1ur7y4fedboh4c3iocfy

Reply:
I don't see anything in that thread or the linked tickets that suggests DMs are not supported.